core/domain/stats_jobs_continuous.py

```diff
@@ -238,12 +269,19 @@ class InteractionAnswerSummariesAggregator(
     """
     @classmethod
     def get_event_types_listened_to(cls):
+        """Returns a list of submitted answer events.
+
+        Returns:
+            list. A list of submitted answer events.
+        """
         return [feconf.EVENT_TYPE_ANSWER_SUBMITTED]
 
     @classmethod
     def _get_realtime_datastore_class(cls):
+        """Returns InteractionAnswerSummariesRealtimeModel class."""
         return InteractionAnswerSummariesRealtimeModel
 
     @classmethod
     def _get_batch_job_manager_class(cls):
+        """Returns InteractionAnswerSummariesMRJobManager class."""
```

Comment:
Add a bit of explanation for the class returned like -

Returns InteractionAnswerSumariesMRJobManager class which calculates interaction view statistics.
ditto for above.

Reply:
Done!